### PR TITLE
workaround for a bug in cuda package dependencies and drivers version…

### PR DESCRIPTION
**Pull request by dtrawins**
… requirement

### CODE REVIEW

**Review by jasiu86 — APPROVED** [review]
looks good

File: Dockerfile.ubuntu [hunk 1]
```diff
@@ -92,20 +121,14 @@ RUN if [ "$NVIDIA" == "1" ] ; then true ; else exit 0 ; fi ; \
     wget -nv https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2004/x86_64/cuda-ubuntu2004.pin -O /etc/apt/preferences.d/cuda-repository-pin-600 ; \
     set -exuo pipefail ; \
     rm -f /etc/apt/apt.conf.d/docker-clean ; \
-    apt-get update && \
-    apt-get install --no-install-recommends -y \
-      gnupg2 \
-      software-properties-common; \
     if [[ ${enable_tensorrt-} == "1" ]] ; then dpkg -i /nv-tensorrt-repo-*.deb ; fi; \
-    apt-key adv --keyserver-options http-proxy=$http_proxy --fetch-keys  https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2004/x86_64/3bf863cc.pub ; \
-    add-apt-repository "deb https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2004/x86_64/ /"; \
```
Comment (mzegla) [inline Dockerfile.ubuntu:101]: Don't we need nvidia repository to download cuda package?

File: Dockerfile.ubuntu [hunk 2]
```diff
@@ -92,20 +121,14 @@ RUN if [ "$NVIDIA" == "1" ] ; then true ; else exit 0 ; fi ; \
...
     apt-get update && apt-get install --no-install-recommends -y \
       libzstd-dev \
-      cuda-11-8 \
       libcudnn8 \
       libcudnn8-dev \
       libcutensor1=1.6.1.5-1 \
       libcutensor-dev=1.6.1.5-1 && \
+    apt-get install --no-install-recommends -y cuda-11-8 && \
```
Comment (mzegla) [inline Dockerfile.ubuntu:131]: Why CUDA installation is extracted here?

**Review by mzegla — APPROVED** [review]

**Review by rasapala — APPROVED** [review]
LGTM

**Pull request merged**